hazelcast/src/test/java/com/hazelcast/scheduledexecutor/AbstractScheduledExecutorNullTest.java

```diff
@@ -0,0 +1,142 @@
+        assertThrowsNPE(s -> s.scheduleOnMember(sampleCallable, sampleMember, 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnMemberAtFixedRate((Runnable) null, sampleMember, 1, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnMemberAtFixedRate(sampleRunnable, null, 1, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnMemberAtFixedRate(sampleRunnable, sampleMember, 1, 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnKeyOwner((Runnable) null, "", 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnKeyOwner(sampleRunnable, null, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnKeyOwner(sampleRunnable, "", 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnKeyOwner((Callable<String>) null, "", 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnKeyOwner(sampleCallable, null, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnKeyOwner(sampleCallable, "", 1, null));
+
+
+        assertThrowsNPE(s -> s.scheduleOnKeyOwnerAtFixedRate((Runnable) null, "", 1, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnKeyOwnerAtFixedRate(sampleRunnable, null, 1, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnKeyOwnerAtFixedRate(sampleRunnable, "", 1, 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnAllMembers((Runnable) null, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnAllMembers(sampleRunnable, 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnAllMembers((Callable<String>) null, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnAllMembers(sampleCallable, 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnAllMembersAtFixedRate((Runnable) null, 1, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnAllMembersAtFixedRate(sampleRunnable, 1, 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnMembers((Runnable) null, sampleMembers, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnMembers(sampleRunnable, null, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnMembers(sampleRunnable, sampleMembers, 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnMembers((Callable<String>) null, sampleMembers, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnMembers((Callable<String>) null, null, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnMembers(sampleCallable, sampleMembers, 1, null));
+
+        assertThrowsNPE(s -> s.scheduleOnMembersAtFixedRate((Runnable) null, sampleMembers, 1, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnMembersAtFixedRate(sampleRunnable, null, 1, 1, sampleTimeUnit));
+        assertThrowsNPE(s -> s.scheduleOnMembersAtFixedRate(sampleRunnable, sampleMembers, 1, 1, null));
+
+        assertThrowsNPE(s -> s.getScheduledFuture(null));
+    }
+
+    private void assertThrowsNPE(ConsumerEx<IScheduledExecutorService> method) {
```

Comment:
Shall we try to use `assertThrows` from `HazelcastTestSupport`?

Reply:
Forgot about that method. Changed, thanks.